Libplanet/Action/AccountStateDeltaImpl.cs

```diff
@@ -16,11 +16,13 @@ namespace Libplanet.Action
     [Pure]
     internal class AccountStateDeltaImpl : IAccountStateDelta
     {
-        private readonly AccountStateGetter _accountStateGetter;
-        private readonly AccountBalanceGetter _accountBalanceGetter;
-        private readonly Address _signer;
-        private IImmutableDictionary<Address, IValue> _updatedStates;
-        private IImmutableDictionary<(Address, Currency), BigInteger> _updatedFungibleAssets;
+#pragma warning disable SA1401
```

Comment:
```suggestion
#pragma warning disable SA1401  // FieldsMustBePrivate
```
Can you please add something like this for code legibility?

Reply:
Okay, I'm rather going to make them properties.

Reply:
Fixed: 46aad246d7d43595daa9b531ed6c5fec96c7a0c2.